internal/locations/locations.go

```diff
@@ -0,0 +1,148 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// Package locations manages base file and directory locations from within the elastic-package config
+package locations
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	elasticPackageDir = ".elastic-package"
+	stackDir          = "stack"
+	packagesDir       = "development"
+
+	temporaryDir = "tmp"
+	deployerDir  = "deployer"
+
+	kubernetesDeployerElasticAgentYmlFile = "elastic-agent.yml"
+	terraformDeployerYmlFile              = "terraform-deployer.yml"
+)
+
+var (
+	serviceLogsDir        = filepath.Join(temporaryDir, "service_logs")
+	kubernetesDeployerDir = filepath.Join(deployerDir, "kubernetes")
+	terraformDeployerDir  = filepath.Join(deployerDir, "terraform")
+)
+
+//LocationManager maintains an instance of a config path location
+type LocationManager struct {
+	StackPath string
+}
+
+// NewLocationManager returns a new manager to track the Configuration dir
+func NewLocationManager() (LocationManager, error) {
```

Comment:
let's preserve consistency with the rest of codebase: `(*LocationManager, error)`. In case of an error, the * LocationManager is nil